--- mysql-workbench-community-8.0.34-src/CMakeLists.txt.orig	2023-07-13 14:52:08.000000000 +0200
+++ mysql-workbench-community-8.0.34-src/CMakeLists.txt	2023-09-16 12:42:01.501693526 +0200
@@ -144,8 +144,7 @@
 check_function_exists (strtoumax HAVE_STRTOUMAX)
 
 set(ANTLR_JAR_FILENAME "antlr-4.13.2-complete.jar")
-get_filename_component(SOURCE_PARENT_DIR ${CMAKE_SOURCE_DIR} DIRECTORY)
-set(LINUX_RES_BIN_DIR ${SOURCE_PARENT_DIR}/linux-res/bin)
+set(LINUX_RES_BIN_DIR ${CMAKE_SOURCE_DIR}/linux-res/bin)
 message("WITH_ANTLR_JAR: ${WITH_ANTLR_JAR}")
 if(WITH_ANTLR_JAR)
   get_filename_component(ANTLR_JAR_USER_DIR ${WITH_ANTLR_JAR} DIRECTORY)
