--- mysql-workbench-gpl-5.2.38-src/library/forms/swig/Makefile.am.wig	2012-03-29 09:35:54.086119577 +0200
+++ mysql-workbench-gpl-5.2.38-src/library/forms/swig/Makefile.am	2012-03-29 09:37:01.652785128 +0200
@@ -2,7 +2,8 @@
 pkglib_LTLIBRARIES=_mforms.la
 
 pkglibdir=$(libdir)/@PACKAGE@/modules
-pkglib_DATA=mforms.py
+pkgdatadir=$(pkglibdir)
+pkgdata_DATA=mforms.py
 
 _mforms_la_SOURCES=\
 mforms_wrap.cxx
--- mysql-workbench-gpl-5.2.38-src/plugins/wb.admin/frontend/Makefile.am.wig	2012-03-29 09:38:10.986117315 +0200
+++ mysql-workbench-gpl-5.2.38-src/plugins/wb.admin/frontend/Makefile.am	2012-03-29 09:38:22.292783795 +0200
@@ -1,7 +1,7 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgdatadir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=\
+pkgdata_DATA=\
     wb_admin_grt.py\
     wb_admin_monitor.py\
     wb_admin_utils.py\
--- mysql-workbench-gpl-5.2.38-src/plugins/wb.bugreport/Makefile.am.wig	2012-03-29 09:38:33.322783615 +0200
+++ mysql-workbench-gpl-5.2.38-src/plugins/wb.bugreport/Makefile.am	2012-03-29 09:38:44.196116768 +0200
@@ -1,7 +1,7 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgdatadir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=\
+pkgdata_DATA=\
   HTMLGetData.py\
   HTMLLookup.py\
   MySQLGetRequestResult.py\
--- mysql-workbench-gpl-5.2.38-src/plugins/wb.doclib/Makefile.am.wig	2012-03-29 09:38:56.102783239 +0200
+++ mysql-workbench-gpl-5.2.38-src/plugins/wb.doclib/Makefile.am	2012-03-29 09:39:08.662783035 +0200
@@ -1,7 +1,7 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgdatadir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=\
+pkgdata_DATA=\
   mysqldoclib.py\
   wb_doclib_grt.py
 
--- mysql-workbench-gpl-5.2.38-src/modules/wb.utils/Makefile.am.wig	2012-03-29 09:37:24.452784753 +0200
+++ mysql-workbench-gpl-5.2.38-src/modules/wb.utils/Makefile.am	2012-03-29 09:37:40.289451158 +0200
@@ -1,6 +1,6 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgdatadir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=catalog_utils.grt.lua  tools.grt.lua  wb_utils_grt.py sqlide_grt.py text_grt.py sql_reformatter.py\
+pkgdata_DATA=catalog_utils.grt.lua  tools.grt.lua  wb_utils_grt.py sqlide_grt.py text_grt.py sql_reformatter.py\
             wb_dev_utils_grt.py table_utils_grt.py code_utils_grt.py
 
