--- mysql-workbench-gpl-5.2.38-src/./frontend/linux/workbench/mysql-workbench.in~	2012-02-21 18:53:41.000000000 +0200
+++ mysql-workbench-gpl-5.2.38-src/./frontend/linux/workbench/mysql-workbench.in	2012-05-11 20:35:21.219997098 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 # Uncomment the following line if you're having trouble with gnome-keyring lockups.
 # This will cause passwords to be stored only temporarily for the session.
--- mysql-workbench-community-6.2.5-src/plugins/wb.admin/backend/wb_server_management.py	2015-05-12 11:40:29.032525721 +0300
+++ mysql-workbench-community-6.2.5-src/plugins/wb.admin/backend/wb_server_management.py	2015-05-12 11:42:48.473107723 +0300
@@ -88,10 +88,10 @@
         if as_user != Users.ADMIN:
             sudo_user = "sudo -u %s" % as_user
             sudo_prefix = sudo_prefix.replace('sudo', sudo_user)
-        if '/bin/sh' in sudo_prefix or '/bin/bash' in sudo_prefix:
+        if '/bin/sh' in sudo_prefix or '/bin/sh' in sudo_prefix:
             command = sudo_prefix + " \"" + command.replace('\\', '\\\\').replace('"', r'\"').replace('$','\\$') + "\""
         else:
-            command = sudo_prefix + " /bin/bash -c \"" + command.replace('\\', '\\\\').replace('"', r'\"').replace('$','\\$') + "\""
+            command = sudo_prefix + " /bin/sh -c \"" + command.replace('\\', '\\\\').replace('"', r'\"').replace('$','\\$') + "\""
 
     return command
 
--- mysql-workbench-community-6.2.5-src/./plugins/migration/wbcopytables.in~	2015-02-25 10:08:47.000000000 +0200
+++ mysql-workbench-community-6.2.5-src/./plugins/migration/wbcopytables.in	2015-05-12 11:45:07.170317749 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 # Set the destdir=<some_dir> when ever you install using DESTDIR=<some_dir>. 
 destdir="$WB_DEST_DIR"
