--- mysql-workbench-gpl-5.2.47-src/./plugins/wb.admin/backend/wb_admin_control.py~	2013-02-14 22:54:03.000000000 +0200
+++ mysql-workbench-gpl-5.2.47-src/./plugins/wb.admin/backend/wb_admin_control.py	2014-05-01 16:06:43.138434870 +0300
@@ -704,7 +704,7 @@
                     self.server_profile.general_log_file_path = general_log_file_path or os.path.join(self.server_profile.datadir, hostname + '.log')
                     request_save_profile = True
 
-                slow_query_log_file = opts[config_section].get('log-slow-queries', '').strip(' "') if opts.has_key(config_section) else ''
+                slow_query_log_file = (opts[config_section].get('log-slow-queries', '') or opts[config_section].get('log_slow_queries', '')).strip(' "') if opts.has_key(config_section) else ''
                 if self.server_profile.slow_log_file_path != slow_query_log_file:
                     self.server_profile.slow_log_file_path = slow_query_log_file or os.path.join(self.server_profile.datadir, hostname + '.slow')
                     request_save_profile = True
