--- mysql-workbench-gpl-5.2.38-src/MySQLWorkbench.desktop.in.wiget	2012-03-29 09:29:49.000000000 +0200
+++ mysql-workbench-gpl-5.2.38-src/MySQLWorkbench.desktop.in	2012-03-29 09:33:26.656122014 +0200
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=MySQL Workbench
 Comment=MySQL Database Design, Administration and Development Tool
-Exec=#bindir#/mysql-workbench
+Exec=mysql-workbench
 Terminal=false
 Type=Application
 Icon=mysql-workbench
