--- mysql-workbench-oss-5.1.9/MySQLWorkbench.desktop.in	2009-03-20 13:40:26.000000000 +0000
+++ mysql-workbench-oss-5.1.9/MySQLWorkbench.desktop.in	2009-04-01 19:30:48.000000000 +0000
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Name=MySQL Workbench
 Comment=MySQL Database Design Tool
-Exec=#bindir#/mysql-workbench
+Exec=mysql-workbench
 Terminal=false
 Type=Application
-Icon=#pkgdatadir#/images/MySQLWorkbench-48.png
-Categories=GTK;Database;Development;Application
+Icon=mysql-workbench
+Categories=GTK;Database;Development;
