--- mysql-workbench-gpl-5.2.34-src/configure.in~	2011-07-14 20:03:23.000000000 +0200
+++ mysql-workbench-gpl-5.2.34-src/configure.in	2011-07-14 20:28:45.034453169 +0200
@@ -68,9 +68,9 @@
   std_edition=$enableval, std_edition=no)
 
 if test x$std_edition = xyes; then
-     echo -en "#ifndef __WB_CONFIG_H__\n#define __WB_CONFIG_H__\n#define EDITION_SE\n#define COMMERCIAL_CODE\n#endif\n" > wb_config.h
+     printf "#ifndef __WB_CONFIG_H__\n#define __WB_CONFIG_H__\n#define EDITION_SE\n#define COMMERCIAL_CODE\n#endif\n" > wb_config.h
 else
-     echo -en "#ifndef __WB_CONFIG_H__\n#define __WB_CONFIG_H__\n#undef COMMERCIAL_CODE\n#define EDITION_OSS\n#endif\n" > wb_config.h
+     printf "#ifndef __WB_CONFIG_H__\n#define __WB_CONFIG_H__\n#undef COMMERCIAL_CODE\n#define EDITION_OSS\n#endif\n" > wb_config.h
 fi
 
 AC_CHECK_HEADERS([sys/select.h])
