http://remi.fedorapeople.org/mysql-workbench-oss-5.1.12-gcc44.patch
diff -up mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_resultset.cpp.gcc44 mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_resultset.cpp
--- mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_resultset.cpp.gcc44	2009-05-01 09:46:32.000000000 +0200
+++ mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_resultset.cpp	2009-05-01 09:46:47.000000000 +0200
@@ -17,6 +17,7 @@
 #include <cppconn/exception.h>
 #include <sstream>
 #include <math.h>
+#include <cstdio> 
 
 
 namespace sql
diff -up mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_rset_metadata.cpp.gcc44 mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_rset_metadata.cpp
--- mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_rset_metadata.cpp.gcc44	2009-05-01 09:44:13.000000000 +0200
+++ mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_art_rset_metadata.cpp	2009-05-01 09:45:41.000000000 +0200
@@ -10,6 +10,7 @@
 */
 
 #include <stdlib.h>
+#include <cstdio> 
 
 #include "mysql_util.h"
 #include "mysql_art_resultset.h"
diff -up mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_resultbind.cpp.gcc44 mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_resultbind.cpp
--- mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_resultbind.cpp.gcc44	2009-04-23 18:00:53.000000000 +0200
+++ mysql-workbench-oss-5.1.12/ext/cppconn/driver/mysql_resultbind.cpp	2009-05-01 09:47:54.000000000 +0200
@@ -14,6 +14,7 @@
 #include "mysql_resultbind.h"
 
 #include <string.h>
+#include <cstdio> 
 
 namespace sql
 {
diff -up mysql-workbench-oss-5.1.12/library/canvas/src/mdc_algorithms.cpp.gcc44 mysql-workbench-oss-5.1.12/library/canvas/src/mdc_algorithms.cpp
--- mysql-workbench-oss-5.1.12/library/canvas/src/mdc_algorithms.cpp.gcc44	2009-05-01 09:49:00.000000000 +0200
+++ mysql-workbench-oss-5.1.12/library/canvas/src/mdc_algorithms.cpp	2009-05-01 09:50:51.000000000 +0200
@@ -1,6 +1,8 @@
+#include <cstdio> 
 #include "stdafx.h"
 #include "mdc_algorithms.h"
 
+
 using namespace mdc;
 
 
diff -up mysql-workbench-oss-5.1.12/library/canvas/src/mdc_area_group.cpp.gcc44 mysql-workbench-oss-5.1.12/library/canvas/src/mdc_area_group.cpp
diff -up mysql-workbench-oss-5.1.12/library/canvas/src/mdc_common.h.gcc44 mysql-workbench-oss-5.1.12/library/canvas/src/mdc_common.h
--- mysql-workbench-oss-5.1.12/library/canvas/src/mdc_common.h.gcc44	2009-05-01 09:54:19.000000000 +0200
+++ mysql-workbench-oss-5.1.12/library/canvas/src/mdc_common.h	2009-05-01 09:53:57.000000000 +0200
@@ -12,6 +12,7 @@
 #include <assert.h>
 #include <algorithm>
 #include <typeinfo>
+#include <cstdio> 
 
 #include <sigc++/sigc++.h>
 
diff -up mysql-workbench-oss-5.1.12/library/forms/filechooser.cpp.gcc44 mysql-workbench-oss-5.1.12/library/forms/filechooser.cpp
--- mysql-workbench-oss-5.1.12/library/forms/filechooser.cpp.gcc44	2009-05-01 09:58:49.000000000 +0200
+++ mysql-workbench-oss-5.1.12/library/forms/filechooser.cpp	2009-05-01 09:59:06.000000000 +0200
@@ -2,6 +2,7 @@
 #include "stdafx.h"
 
 #include <mforms/mforms.h>
+#include <cstdio> 
 
 
 using namespace mforms;
diff -up mysql-workbench-oss-5.1.12/library/forms/mforms.cpp.gcc44 mysql-workbench-oss-5.1.12/library/forms/mforms.cpp
--- mysql-workbench-oss-5.1.12/library/forms/mforms.cpp.gcc44	2009-05-01 10:00:27.000000000 +0200
+++ mysql-workbench-oss-5.1.12/library/forms/mforms.cpp	2009-05-01 10:00:40.000000000 +0200
@@ -3,6 +3,7 @@
 #include <string.h>
 #include "mforms/mforms.h"
 #include "mforms/simpleform.h"
+#include <cstdio> 
 
 using namespace mforms;
 
