--- mysql-workbench-gpl-5.2.26/plugins/db.mysql/frontend/db_reverse_engineer.cpp	2010-08-05 13:33:19.000000000 +0000
+++ mysql-workbench-gpl-5.2.26/plugins/db.mysql/frontend/db_reverse_engineer.cpp	2010-09-16 20:52:20.597866900 +0000
@@ -373,13 +373,6 @@
       int total_placable_object_count= plugin->db_objects_selection_model(Db_plugin::dbotTable)->active_items_count() +
         plugin->db_objects_selection_model(Db_plugin::dbotView)->active_items_count() +
         plugin->db_objects_selection_model(Db_plugin::dbotRoutine)->active_items_count();
-      if (total_placable_object_count > 250)
-      {
-        mforms::Utilities::show_warning(_("Resource Warning"), _("Too many objects are selected for auto placement."
-          "\nSelect fewer elements to create the EER diagram."), "OK");
-        _autoplace_check.set_active(false);
-        return false;
-      }
     }
 
     if (!text.empty())
