--- mysql-workbench-community-6.3.10-src/CMakeLists.txt~	2019-11-02 23:52:15.000000000 +0100
+++ mysql-workbench-community-6.3.10-src/CMakeLists.txt	2019-11-02 23:52:44.896265071 +0100
@@ -288,8 +288,8 @@
 
 # Add some common compiler flags. TODO: Shouldn't this be done checking if whatever compiler
 # we use supports each flag to allow for non-gcc compilers in Linux (as above)?
-set(WB_CXXFLAGS -Werror -Wall -Wextra -Wno-unused-parameter -Wno-deprecated -std=c++11)
-set(WB_CFLAGS -Werror -Wall -Wextra -Wno-unused-parameter -Wno-deprecated)
+set(WB_CXXFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-deprecated -std=c++11)
+set(WB_CFLAGS -Wall -Wextra -Wno-unused-parameter -Wno-deprecated)
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
   if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.7")
     message(FATAL_ERROR "Insufficient gcc version. You need gcc v4.7 or newer to build this project")
