--- mysql-workbench-gpl-5.2.38-src/res/mysql.profiles/Makefile.am~	2012-02-21 18:53:45.000000000 +0200
+++ mysql-workbench-gpl-5.2.38-src/res/mysql.profiles/Makefile.am	2012-05-11 11:46:32.197808250 +0300
@@ -9,6 +9,7 @@
 Linux_(Custom).xml\
 MacOS_X_(MySQL_Package).xml\
 OpenSolaris_(MySQL_Package).xml\
+PLD_Linux_(MySQL_Package).xml \
 RHEL_(MySQL_Package).xml\
 SLES_(MySQL_Package).xml\
 Ubuntu_Linux_(MySQL_Package).xml\
