--- mysql-workbench-community-8.0.34-src/library/cdbc/src/driver_manager.h~	2023-07-13 14:52:08.000000000 +0200
+++ mysql-workbench-community-8.0.34-src/library/cdbc/src/driver_manager.h	2023-09-16 16:49:38.679479295 +0200
@@ -27,6 +27,7 @@
 #include "cppdbc_public_interface.h"
 
 #include <cppconn/driver.h>
+#include <cstdint>
 #include <memory>
 #include <set>
 
--- mysql-workbench-community-8.0.34-src/library/forms/mforms/treeview.h~	2023-07-13 14:52:08.000000000 +0200
+++ mysql-workbench-community-8.0.34-src/library/forms/mforms/treeview.h	2023-09-16 16:52:45.258483636 +0200
@@ -23,6 +23,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <mforms/view.h>
 
 /**
--- mysql-workbench-community-8.0.34-src/backend/wbpublic/sqlide/sqlide_generics_private.h~	2023-07-13 14:52:08.000000000 +0200
+++ mysql-workbench-community-8.0.34-src/backend/wbpublic/sqlide/sqlide_generics_private.h	2023-09-16 16:59:52.762837702 +0200
@@ -30,6 +30,7 @@
 #include <sqlite/query.hpp>
 #include <sqlite/execute.hpp>
 #include <ctime>
+#include <cstdint>
 
 namespace sqlide {
 
